ql/src/java/org/apache/hadoop/hive/ql/scheduled/ScheduledQueryExecutionService.java

```diff
@@ -223,6 +223,10 @@ private void processQuery(ScheduledQueryPollResponse q) {
         HiveConf conf = new HiveConf(context.conf);
         conf.set(Constants.HIVE_QUERY_EXCLUSIVE_LOCK, lockNameFor(q.getScheduleKey()));
         conf.setVar(HiveConf.ConfVars.HIVE_AUTHENTICATOR_MANAGER, SessionStateUserAuthenticator.class.getName());
+        conf.set(Constants.SCHEDULED_QUERY_NAMESPACE, q.getScheduleKey().getClusterNamespace());
```

Comment:
When is unsetClusterNamespace() called? Does the clusterNamespace get impacted in that case?

Reply:
This is part of Zoltan's patch. Not sure what the question is

Reply:
We are relying  on conf.set(Constants.SCHEDULED_QUERY_NAMESPACE, q.getScheduleKey().getClusterNamespace()); With that context I am asking when is unsetClusterNamespace() called.

Reply:
Its not unset

Reply:
Not the unset for conf conf.set(Constants.SCHEDULED_QUERY_NAMESPACE). What I am trying to find out is what happens if 'unsetClusterNamespace()' has been called from different flow and then this code 
conf.set(Constants.SCHEDULED_QUERY_NAMESPACE, q.getScheduleKey().getClusterNamespace()); is called. Is that a possibility?